src/main/io/serial_msp.c

```diff
@@ -901,6 +904,14 @@ static bool processOutCommand(uint8_t cmdMSP)
         serialize8(masterConfig.auto_disarm_delay); 
         serialize8(masterConfig.disarm_kill_switch);
         break;
+    case MSP_FAILSAFE_CONFIG:
+        headSerialReply(8);
+        serialize8(masterConfig.failsafeConfig.failsafe_delay);
+        serialize8(masterConfig.failsafeConfig.failsafe_off_delay);
+        serialize16(masterConfig.failsafeConfig.failsafe_throttle);//also in MSP_MISC, just to keep things in sync for now
+        serialize16(masterConfig.rxConfig.rx_min_usec);
+        serialize16(masterConfig.rxConfig.rx_max_usec);                                
```

Comment:
IMHO rx_min_usec and rx_max_usec should be added to MSP_RX_CONFIG / MSP_SET_RX_CONFIG and not failsafe config.   If you add them to the end you can retain backwards compatibility in MSP_SET_RX_CONFIG by only reading/setting the values depending on the data length.